wiki/Medals/en.md

```diff
@@ -1,59 +1,295 @@
-- The beatmap can be deleted safely once the **`CLEARED`** status was noted in the [Beatmap Pack Listing](https://osu.ppy.sh/beatmaps/packs?type=theme).
+It is worth noting however that the contents of all the Video Game Packs, Anime Packs, Internet! Packs, and Rhythm Game Packs have been dramatically altered over time since [their first introduction in 2008](https://osu.ppy.sh/community/forums/topics/1853). For the list of all the beatmaps that was being listed in the aforementioned beatmap packs in their legacy/original state, see [Legacy Beatmap Packs](/wiki/Medals/Legacy_Beatmap_Packs).
 
 ## Skill
 
-The **Skill** medals are currently divided into three groups: Combo, High Ranker, and Star-based.
+The **Skill medals** are medals that are being awarded by rounding off various skill-based accomplishments, be it by getting into certain [combo](/wiki/en/Glossary/Combo_%28score_multiplier%29) milestones, by attaining certain ranks in the [global performance point leaderboard](https://osu.ppy.sh/rankings/osu/performance), or by achieving either a clear or a [full combo](/wiki/FC) on any [Qualified](/wiki/Beatmap/Category#qualified), [Ranked](/wiki/Beatmap/Category#ranked), or [Approved](/wiki/Beatmap/Category#approved) beatmap within a certain [star rating](https://osu.ppy.sh/wiki/en/Beatmap/Difficulty#star-rating) ranges.
 
-The Combo medals can be obtained by gaining a specific [combo](/wiki/combo) in osu!standard and can be completed with any one song. Achieving the same combo count in [osu!taiko](/wiki/osu!taiko), [osu!catch](/wiki/osu!catch), or [osu!mania](/wiki/osu!mania) mode will not result in getting the medal.
+### List of obtainable Skill medals
 
-For the 1,000 and 2,000 combo medals, you should look for sufficiently long beatmaps. **If the beatmap is not in the Ranked or Approved state, the medals will not be granted.**
+| Medal name | Requirement(s) |
+| :-- | :-- |
+| 500 Combo | Achieve 500 combo in any beatmap, in any game mode. |
+| 750 Combo | Achieve 750 combo in any beatmap, in any game mode. |
+| 1000 Combo | Achieve 1000 combo in any beatmap, in any game mode. |
+| 2000 Combo | Achieve 2000 combo in any beatmap, in any game mode. |
+| I can see the top | Achieve a Global Ranking of #50,000 or higher in any game mode. |
+| The gradual rise | Achieve a Global Ranking of #10,000 or higher in any game mode. |
+| Scaling up | Achieve a Global Ranking of #5,000 or higher in any game mode. |
+| Approaching the summit | Achieve a Global Ranking of #1,000 or higher in any game mode. |
+| Rising Star (osu!)/My First Don (osu!taiko)/A Slice Of Life (osu!catch)/First Steps (osu!mania) | Clear any Qualified, Ranked, or Approved beatmap that lies in the 1.00★–1.99★ star rating range (after all the [difficulty increase mods](/wiki/Game_modifier#difficulty-increase), if any, have been factored in) in the corresponding game modes. Any usage of [difficulty reduction mods](/wiki/Game_modifier#difficulty-reduction) is not permitted. |
```

Comment:
never actually thought about using the unicode(?) star to denote star ratings, i wonder if this should be standard or not 🤔